src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import { useCallback, useEffect, useState } from "react"
 
 interface AddressVerificationFlowProps {
-  verifiedAddressResult: AddressVerificationFlow_verifiedAddressResult$data
+  verifyAddress: AddressVerificationFlow_verifyAddress$data
```

Comment:
I renamed this while trying to get the test wrapper to render this component. I observed that it was expecting the name of the prop to match the name of the GraphQL query, so I assumed that was a convention and it worked.